split.py

```diff
@@ -0,0 +1,31 @@
+from sklearn.model_selection import train_test_split
+import pickle as pkl
+import pandas as pd
+from pathlib import Path
+
+import config
+import utils
+
+# Create split_data directory if it doesn’t exist
+Path(config.SPLIT_DATA_DIRECTORY_PATH).mkdir(parents=True, exist_ok=True)
+
+# Load processed data
+with open(config.EMOTIFY_SPECTROGRAM_DUMP_PATH, "rb") as f:
+    all_spectro = pkl.load(f)
+
+with open(config.EMOTIFY_LABELS_DUMP_PATH, "rb") as f:
+    labels = pkl.load(f)
+
+# Transform labels to pandas DataFrame (mainly to use idxmax)
+labels = pd.DataFrame(labels)
+labels.columns = config.EMOTIFY_EMOTIONS_ORDERED_LIST
```

Comment:
It would be nice to write this in a function so we can call it every time we need a `labels` dataframe. Can be done in another PR.